core/src/main/java/com/linecorp/armeria/server/docs/DocService.java

```diff
@@ -333,8 +333,8 @@ private ServiceInfo addServiceExamples(ServiceInfo service) {
                         // generated by the plugin.
                         concatAndDedup(exampleHeaders.get(m.name()), m.exampleHeaders()),
                         concatAndDedup(exampleRequests.get(m.name()), m.exampleRequests()),
-                        examplePaths.get(m.name()),
-                        exampleQueries.get(m.name()),
+                        concatAndDedup(examplePaths.get(m.name()), m.examplePaths()),
+                        concatAndDedup(exampleQueries.get(m.name()), m.exampleQueries()),
```

Comment:
one question, in this PR I remove the `ImmutableList.builderWithExpectedSize(Iterables.size(examplePaths));` and change it don't use size. Because below code is making iterable after strmead.
```
    private static <T> Iterable<T> concatAndDedup(Iterable<T> first, Iterable<T> second) {
        return Stream.concat(Streams.stream(first), Streams.stream(second)).distinct()::iterator;
        return Stream.concat(Streams.stream(first), Streams.stream(second)).distinct()::iterator;
    }
}
```

Is it intentioned? I mean when we see Iterable parameter, should we think we just can iterate once.

Reply:
Oops I think we can just do `distinct().collect(toImmutableList());`